pwa/src/pages/entities/[entityId]/objects/[objectId].tsx

```diff
@@ -37,7 +37,7 @@ const IndexPage = (props) => {
                 aria-labelledby="overview-tab"
               >
                 <br />
-                <ObjectEntityFormNew entityId={entityId} objectEntityId={objectEntityId} />
+                <ObjectEntityFormNew entityId={entityId} objectId={objectId} />
```

Comment:
We could spread these props, like so: `<ObjectEntityFormNew {...{entityId, objectId}} />`